### PR TITLE
Add email queue logging capability

### PR DESCRIPTION
**Pull request by elvetemedve**
In particular cases Magento sends emails asynchronously via email queues.
This commit extends the current logging functionality to cover queued emails as well.

### CODE REVIEW

File: src/Model/Email/Queue/Logger.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+        if ($parameters['is_plain']) {
+            $log->setBodyText($queue->getMessageBody());
+        } else {
+            $log->setBodyHtml($queue->getMessageBody());
+        }
+        $log->setRecipients($queue->getRecipients());
+        $log->setSubject($parameters['subject']);
+        $log->setFromEmail($parameters['from_email']);
+        $log->setFromName( $parameters['from_name']);
```
Comment (technodelight) [inline src/Model/Email/Queue/Logger.php]: I've spot a blank space character there! :) Could you please remove?

File: src/Model/Email/Queue/Logger.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        $log = $this->_covertEmailQueueToLog($queue);
+        $log->save();
+    }
+
+    /**
+     * @param Mage_Core_Model_Email_Queue $queue
+     *
+     * @return Zenc_EmailLogger_Model_Log
+     */
+    private function _covertEmailQueueToLog($queue)
```
Comment (technodelight) [inline src/Model/Email/Queue/Logger.php]: Could you add the same typehinting here as the above method has?

**Comment by elvetemedve** [comment]
@technodelight I've fixed both issues.

**Comment by technodelight** [comment]
smooth
fatality

**Pull request merged**